src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -40,6 +40,11 @@ public interface IPreferences
         /// </summary>
         bool IsBackgroundGridVisible { get; set; }
 
+        /// <summary>
+        /// Indicates the Default Geometry Scaling value from the Preferences panel
+        /// </summary>
+        double DefaultScaleFactor { get; set; }
```

Comment:
@zeusongit this is an API break. It should not have been merged to 2.18, it needs to be fixed.

Reply:
Agreed @mjkkirschner - we can slip to 2.18 if it's a new feature.

Reply:
@mjkkirschner for this case I can revert this change and remove this property but I think the objective is serialize the Default Scale Factor value in the DynamoSettings.xml file (so it can be also read when re-opening Dynamo), do you know if there is any other way to do it?
Thanks

Reply:
yes, just don't modify the interface, modify the concrete class.

Reply:
Fixed in the next PR, just removed from IPreferences.
https://github.com/DynamoDS/Dynamo/pull/13734